extensions/wikia/AssetsManager/config.php

```diff
@@ -208,6 +208,7 @@
 		'//extensions/wikia/AdEngine/js/utils/math.js',
 		'//extensions/wikia/AdEngine/js/utils/time.js',
 		'//extensions/wikia/AdEngine/js/video/articleVideoAd.js',
+		'//extensions/wikia/ArticleVideo/scripts/featured-video.cookies.js',
```

Comment:
Do we need to add this asset on all pages? I believe it's already available on FV pages. We can use require.optional in playerTracker.js so it won't throw errors on non-FV pages